Tooling.md

```diff
@@ -44,6 +44,12 @@ The tooling we expect to support includes:
   - Instrumentation-based.
 * Process dump: local variables, call stack, heap, global variables, list of
   threads.
+* JavaScript+WebAssembly size optimization tool. Some developers have huge
```

Comment:
Good point. I updated the sentence, and shortened it so it fits better with the surrounding.